tests/unit_tests/models/test_base_run_model.py

```diff
@@ -115,28 +116,11 @@ def get_run_path_mock(realizations, iteration=None):
             return [f"out/realization-{r}/iter-{iteration}" for r in realizations]
         return [f"out/realization-{r}" for r in realizations]
 
-    brm = BaseRunModel(simulation_arguments, None, None, None)
+    brm = BaseRunModel(simulation_arguments, None, None, None, None)
     brm.facade = MagicMock(
         run_path=run_path,
         number_of_iterations=number_of_iterations,
         get_run_paths=get_run_path_mock,
     )
 
     assert brm.check_if_runpath_exists() == expected
-
-
-def test_validation():
```

Comment:
Why is this test removed?

Reply:
Validation has been removed in this PR as a lot of validations are for writing into existing storages, something that this PR doesn't allow.